src/style.css

```diff
@@ -0,0 +1,45 @@
+body {
+  margin: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-image: url(images/star.svg);
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 :first-child {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 :not(:nth-child(n+3)) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 :not(:nth-child(n+4)) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 :not(:nth-child(n+5)) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 :nth-child(n) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  margin: 0;
+  display: flex;
+}
```

Comment:
There are a few issues to fix:
- you didn't use class `stars`, that's better to add a flexbox there but not to customize `start--...`
- duplicating styles when setting `background-image`, try to combine it as you did down below

As a reference, you can review solutions of your peers